font/src/ft/mod.rs

```diff
@@ -225,13 +243,14 @@ impl FreeTypeRasterizer {
                 None
             };
 
+            let load_flags = Self::ft_load_flags(pattern);
             let face = Face {
                 ft_face: ft_face,
                 key: FontKey::next(),
-                load_flags: Self::ft_load_flags(pattern),
+                load_flags: load_flags,
                 render_mode: Self::ft_render_mode(pattern),
                 lcd_filter: Self::ft_lcd_filter(pattern),
-                non_scalable: non_scalable,
+                non_scalable: non_scalable
```

Comment:
Tiny nitpick but these changes from line 246 to line 253 are not necessary, right? At least I can't see why this got changed.

Reply:
Yeah, those were just screw-ups I forgot to clean up. Fixing.